docs/guides/how_to_guides/creating_and_editing_expectations/how_to_create_an_expectation_suite_with_the_user_configurable_profiler.rst

```diff
@@ -242,9 +242,369 @@ This guide will help you create a new Expectation Suite by profiling your data w
+                        class_name: DefaultExpectationConfigurationBuilder # Optional, will use Default if not supplied, otherwise will use supplied
+                        module_name: tbd # Optional, for all other
+                        expectation_type: expect_column_values_to_match_strftime_format # QUESTION: Should we change this from expectation to expectation_type?
+                        column: $my_domain_builder.domain_kwargs.column  # is this obvious/inferrable?
+                                                              # AJB - I think some may wish to be more explicit but yes
+                                                              # this should be inferrable when there is a 1-1 between
+                                                              # ProfilerRule and Domain Builder. So make this line optional.
+                        strftime_format: $my_dateformat.date_format_string # This is an expectation kwarg
+                                                                                     # (Parameters in our docs)
+                                                                                     # QUESTION: removed `parameter`
+                - branch:
+                    if: $domain.column_type in my_dateformat.DATETIME_TYPES_CONST  # Note that domain returned "column_type" in addition to "domain_kwargs" - Note also that DATEITME_TYPES_CONST doesn't yet exist but would be a class variable of the SimpleDateFormatStringParameterBuilder
+                                                                                   # QUESTION: Is this a "subdomain"? It kind of feels against the 1 domain per ProfilerRule, but makes sense. Suppose I want to reference this elsewhere, I guess for now we copy paste.
+                    then:
+                      - expectation: expect_column_values_to_be_in_type_list
+                        column: $domain_builder.domain_kwargs.column  # is this obvious/inferrable? # YES optional, see above
+                        type_list: my_dateformat.DATETIME_TYPES_CONST # See above note
+            my_rule_for_numerics:  # "just-a-name"
+              # Which columns in this data are numeric?
+              class_name: SemanticTypeColumnDomainBuilder
+              semantic_types:
+                - numeric
+              parameter_builders:
+                - parameter_name: my_parameter_mean
+                  class_name: MetricParameterBuilder
+                  metric_name: column.mean
+                  metric_domain_kwargs: $domain.domain_kwargs # QUESTION: Should this be optional and inferred?
+                - parameter_name: my_parameter_min
+                  class_name: MetricParameterBuilder
+                  metric_name: column.min
+                  metric_domain_kwargs: $domain.domain_kwargs # QUESTION: Should this be optional and inferred?
+                - parameter_name: my_parameter_false_positive_threshold_min
```

Comment:
Could we name this something more intuitive to insinuate that this is a multibatch parameter?